lib/ansible/modules/cloud/azure/azure_rm_networkinterface.py

```diff
@@ -59,6 +59,12 @@
             - virtual_network
         required: true
         default: null
+    virtual_network_resource_group:
+        description:
+            - Resource group of the virtual network. Use current resource group as default if this field not specificed.
+        required: false
```

Comment:
`required: false` is the default and is easier to just omit it in the documentation